ledger/src/blockstore.rs

```diff
@@ -1764,6 +1767,28 @@ impl Blockstore {
         }
     }
 
+    pub fn get_block_height(&self, slot: Slot) -> Result<Option<u64>> {
+        datapoint_info!(
+            "blockstore-rpc-api",
+            ("method", "get_block_height".to_string(), String)
+        );
+        let lowest_cleanup_slot = self.lowest_cleanup_slot.read().unwrap();
+        // lowest_cleanup_slot is the last slot that was not cleaned up by
+        // LedgerCleanupService
+        if *lowest_cleanup_slot > 0 && *lowest_cleanup_slot >= slot {
```

Comment:
lol beginning to realize this should probably be called `highest_cleanup_slot`

Reply:
Oh geez, 💯 
A fix for another PR, though.